outputs.tf

```diff
@@ -34,11 +34,13 @@ output "zone_id" {
 output "listeners" {
   description = "Map of listeners created and their attributes"
   value       = aws_lb_listener.this
+  sensitive   = true
```

Comment:
adding it here is a bit "heavy handed" because it marks all of the output as sensitive, even when users have not supplied any sensitive details. Its better to handle this in your implementation when creating outputs from the module